sos/report/plugins/foreman_proxy.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (C) 2015 Red Hat, Inc., Pavel Moravec <pmoravec@redhat.com>
```

Comment:
2021? ;-)

Reply:
OK, now I updated the name from copy&paste, there is *always* something I can forget on.. :)